plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/RetryDriver.java

```diff
@@ -41,6 +42,7 @@ public class RetryDriver
     private final double scaleFactor;
     private final Duration maxRetryTime;
     private final List<Class<? extends Exception>> stopOnExceptions;
+    private final AtomicInteger attempt = new AtomicInteger(0);
```

Comment:
was RetryDriver immutable until now?

Reply:
I think so. Do we want to move this counter to `ThriftHiveMetastore#dropTable`?

Reply:
Changed